configuration/rules-dsl.md

```diff
@@ -301,6 +300,30 @@ then
 end
 ```
 
+{: #transformations}
+### Transformations
+
+openHAB [Transformation services]({{base}}/addons/transformations.html) can also be used in rules to **transform/translate/convert data**.
+The general syntax is as follows:
+
+```java
+transform("<transformation-identifier>", "<transf. expression or transf. file name>", "<input-data or variable>")
+```
+
+- `<transformation-identifier>` - Shorthand identifier of the transformation service
+- `<transf. expression or transf. file name>` - Transformation service specific
+- `<input-data or variable>` - The data to transform, MUST be of data type *String*
```

Comment:
Very clear!  I love the capital letters MUST.  Keeps someone from glossing over this point, and potentially working for hours to fix an incorrectly set data type.

Reply:
:)